website/package.json

```diff
@@ -11,7 +11,7 @@
     "@hashicorp/react-alert-banner": "6.1.1",
     "@hashicorp/react-button": "5.0.0",
     "@hashicorp/react-content": "7.0.1",
-    "@hashicorp/react-docs-page": "13.0.3",
+    "@hashicorp/react-docs-page": "13.1.7-alpha.9",
```

Comment:
Will bump this to a stable release once these PRs are finalized and https://github.com/hashicorp/react-components/pull/220 is merged.

Reply:
✅ Bumped to stable